src/server/scripts/Northrend/VioletHold/instance_violet_hold.cpp

```diff
@@ -199,16 +201,21 @@ class instance_violet_hold : public InstanceMapScript
                     break;
                 default:
                     break;
+                case NPC_VOID_SENTRY:
+                    if (zuramatDead)
+                    {
+                        creature->DespawnOrUnsummon();
+                        zuramatDead = false;
```

Comment:
So if (hypothetically) there are two sentries spawned after he dies only one will be despawned? (why is this being set to false?)